README.md

```diff
@@ -18,3 +18,9 @@ gulp
 ```
 
 See [Build Details](https://github.com/FullScreenShenanigans/Documentation/blob/master/Build%20Details.md) for detailed Gulp usage.
+
+
+## Adding Functionality
```

Comment:
This section is inaccurate and shouldn't be in the readme. Please remove.

See https://github.com/FullScreenShenanigans/Documentation/blob/master/Build%20Details.md.